src/otx/algo/detection/heads/base_head.py

```diff
@@ -74,7 +74,6 @@ def __init__(self, init_cfg: ConfigDict | list[ConfigDict] | dict | list[dict] |
 
     def init_weights(self) -> None:
         """Initialize the weights."""
-        super().init_weights()
```

Comment:
It's just a question. Why is it removed?

Reply:
Original base_head in mmdet inherits mmengine's base module and it has init_weights methods.
However base_head in otx inherits nn.Module and it don't have init_weights methods